README.md

```diff
@@ -1,4 +1,4 @@
-CloundFront link https://d3ei0f9jfo5y8p.cloudfront.net
+CloundFront link https://d3j4uh2paptzis.cloudfront.net/
 S3 link https://aws-ep-vs.s3.amazonaws.com/index.html
```

Comment:
can you please add in PR description - what are the additions tasks that you have covered?